src/routes/html.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+
+const router = express.Router()
+const documentation = require('../controllers/documentation')
+const home = require('../controllers/home')
+
+router.get('*', (req, res) => {
+  const params = req.originalUrl.split('?')[0].split('/')
+  switch (params[1]) {
+  case '':
+    res.send(home())
+    break
+  case 'health':
+  case 'healthcheck':
+    res.send('')
```

Comment:
what is this used for?  not yet implemented?

Reply:
For monitoring software to check if server is alive, page is empty but they just checking for status code 200.